subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[jvm_test_suite_plugin]]
+= The JVM Test Suite Plugin
+
+The JVM Test Suite plugin (plugin id: `jvm-test-suite`) allows for defining multiple groups of automated tests in a Java project.  These tests can be compiled once and run multiple times against various different target environments (for example, on different JVMs).
+
+This plugin can be used to define a group of Integration Tests, which might run much longer than unit tests and have different environmental requirements, and ought to be run against all the different JVMs which will be supported in production.
```

Comment:
We need to soften this because the different JVMs/target environments is aspirational in this release. We don't yet have an easy way for someone to declare they want to run their tests with multiple JDKs.

What I would focus on is that the test suite plugin is a way to easily model new groups of tests and leave out that you will eventually be able to target multiple environments.